Makefile

```diff
@@ -1,11 +1,26 @@
-all: generate test lint
+SHELL := /usr/bin/env bash
+
+GOGC ?= 1000 # Reduce GC frequency during testing, default to 1000 if unset.
+
+all: generate test fuzz lint
 
-test: GOGC ?= 1000 # Reduce the GC frequency, default to 1000 if unset.
 test:
 	GOGC=$(GOGC) go test ./...
 .PHONY: test
 
+fuzz: FUZZTIME ?= 10s # The duration to run fuzz testing, default to 10s if unset.
+fuzz: # List all fuzz tests across the repo, and run them one at a time with the configured fuzztime.
+	@go run ./scripts/list_fuzz_tests | while read -r line; do \
```

Comment:
We can write this as:

```bash
go list ./... | while read package; do \
	go test -list '^Fuzz' "$$package" | while read func; do \
        echo "Running ./$$package $$func for $(FUZZTIME)..."; \
        GOGC=$(GOGC) go test ./$$package -fuzz=$$func -fuzztime=$(FUZZTIME); \
    done
done
```

Then we don't need the go program.

Reply:
That's much nicer thanks Steven

Reply:
I was going to suggest this 😄